src/components/ScrollToTop.js

```diff
@@ -0,0 +1,17 @@
+// import { useEffect } from 'react';
+// import { withRouter } from 'react-router-dom';
+
+// function ScrollToTop({ history }) {
+//   useEffect(() => {
+//     const unlisten = history.listen(() => {
+//       window.scrollTo(0, 0);
+//     });
+//     return () => {
+//       unlisten();
+//     }
+//   }, []);
+
+//   return (null);
+// }
+
+// export default withRouter(ScrollToTop);
```

Comment:
```suggestion
```
You can always exclude this file when commiting your changes to github, so it doesn't appear in the pull request :)